gateways/istio-ingress/values.yaml

```diff
@@ -11,6 +11,9 @@ gateways:
     - port: 80
       targetPort: 80
       name: http2
+    - port: 15029
```

Comment:
Any reason for adding another port?

In any case - don't set targetPort, prevents ingress installing in multiple namespaces.

Reply:
I mean: we can expose all services on the same port - either 443 or 15443, I don't think we want one port per component.

Reply:
added for testing before, remove it now